### DISCUSSION TITLE
Feelings and thoughts about the current seasonal race format

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#4484] [General]
Hello! I would like to ask people how they FEEL about the current seasonal race format. This includes Tickets, Deliveries, Chores, Auctions, Stella Items and other features related to the season.

**What is it that you are feeling regarding seasonal races? E.g. boredom, frustration, repetitiveness etc.? Is the season too long/stressful? Do you classify yourself as a top-end player gunning for the top auctions or a casual laid-back player just going after Stella items?**

The team is open to entertain the idea of changing things up for the season. This can come in a multitude of changes:
- Remove Faction Leaderboard Tickets
- Change the chore structure to weekly and make them overloaded (No limit on chores for the week)
- Make the seasonal race shorter
- Break up the season in multiple stages. Month 1: Seasonal tickets for auctions. Month 2: Stella items. Month 3: Other/resource auction/dynamic mint
- Other options

Not looking for suggestions in this thread, mainly looking at feelings and reasons as to why you don't participate in x or feel stressed in z etc. :) Remember to keep it civil, constructive and no snarkyness. it will be listened to more <3

**Comment by jaysuper711** [comment]
I don't like that the seasonal tickets are included in the faction leaderboard. Being in the top 10 on the leaderboard has made me feel like I have to continue to donate to the faction or lose out on one of the top items. I would like to be able to choose which week to participate in rather than feeling obligated to. I do like the weekly faction chores, but I find I run through them fairly quickly and it is a long time before they get refreshed.

**Comment by ShinKan42** [comment]
- I think Faction Ticket is activating the market, which is a great thing. Should have something to compensate for removing it, or else the market would be way dead and trading would not be an option to play.// I'm pretty much enjoying this which gives the game a strategy to play
- Weekly chores definitely great!
- Multiple stages are great. But should consider making it still attractive. The Auction is driving the whole season feature for now.

Perhaps we can have more ways to mint so tickets won't be the MacGuffin everyone would only look for.

**Comment by dragonhunter2017** [comment]
Hi Decol,
thanks for asking. I consider myself as a dedicated player aiming to get the top tier 1 or 2 items through tickets and buy some stella items from others through opensea. I have a life time banner. But unfortunately, I have stopped participating this season.

- I am playing from europe and the leaderboard reset at 2:00 midnight. What I have seen that I went to sleep with a position 22 and in the morning found that I ended up at 27 with a few marks difference which is really frustrating. Even if i buy a lot of resources from market, there is no guarantee that I will be in a good position after reset. So there is no chance for me to get any of the top items.

-  top items are only possible to get by large bb or if you have to be in the top position in faction leaderboard. the rest items in auction or in stella are mostly useless, I will say. They are reducing lemon/tomato growth time by 25 or 50 % and you have only 12 seeds/restock. The one in stella store with 5k is give +1 tomato, which is also not sth to go for. I would rather like to see one of the top tier item , ex. gnome pharaoh or oil overall for 5k ticket, then I can see a reason to collect tickets.

- I don't test my luck, so I am not going to waste my time for treasure key-->ticket or this kind of mambo-jambo.

only positive thing, is the faction marks shop. so I am collecting marks and I know that at the end I will get some cool nft.

**Reply by ShinKan42** [reply]
yea, i pretty quit going for 25 and keep 50 instead

**Comment by Dionis404** [comment]
Starting with kraken season, I started chasing 1st-2nd levels items.
This season I'm going for Gnome because that's the limit I can pick up, Shovel or Drill can only be picked up by taking the top positions in the faction.

I have slightly mixed feelings about the fact that OP items can only get top players thus they will constantly be top players.

This race feels more stressful due to the amount of deliveries and activities that need to be done.
Ideas: 
1. Reduce the number of deliveries. Example: If you give delivery to Tywin, delivery to Cornwell becomes unavailable.
2. Limit minigames every day to 3 games takes a lot of time. Give a reward for only one minigame.
3. Change the Henk`s assignment system. weekly assignments with choice assignments will allow players to play more relaxed.
4. Limit the surrender of resources to the faction. As written above. At night, many players throw you off your place in the leaderboard. 
As an option, you could make a daily limit of 20-30 for each position. Then players will stop emphasizing on the cheapest resource and will try to pass all 3 positions in the smithy and 3 positions of the dish to the pet to get more marks.

I also miss the seasonal mechanics that would affect race participation. Like it was a maze or kraken catch.

**Reply by ShinKan42** [reply]
I don't think the leaderboard is about the donation limit.
Limit won't solve the Timezone issue.
Ppl will still find themselves lagging after sleeping. Or else the later timezone player will find themselves impossible to catch up since they came the last.

**Reply by Dionis404** [reply]
Simple calculations: 
On the day, you have 6 change options for stamps 3 kitchen and 3 pet
We have limited each of these options to 20. 

6*20 = 120 times you can give each day.

Accordingly, we get a weekly limit and a lot will depend on the items and rank.

**Reply by ShinKan42** [reply]
> Simple calculations: On the day, you have 6 change options for stamps 3 kitchen and 3 pet We have limited each of these options to 20.
> 
> 6*20 = 120 times you can give each day.
> 
> Accordingly, we get a weekly limit and a lot will depend on the items and rank.

I mean:
①players in different timezones will still climb the ladder at different times which cannot solve
 the "surprised" problem mentioned above.
②less demand will have a less active economy and less fun for traders like me. I've find it a fun play style and I think we need to have some other ways to solve it rather than simple capping.

**Reply by Dionis404** [reply]
Now players just buy the cheapest resource of the week and donate it non-stop in the last minutes. 

In the case of restrictions you will realize that you have passed the maximum and if you will be removed it will be fair because you have passed your maximum

**Reply by Dionis404** [reply]
If you have passed all 840 requests in a week and you are overtaken then your rank or clothes are too weak, and for complete fairness in case of the same values you can sort the list by experience (like at the auction). 
Such a system will stimulate the economy to a greater extent because players will have to turn in as many requests as possible.

**Reply by ShinKan42** [reply]
> If you have passed all 840 requests in a week and you are overtaken then your rank or clothes are too weak, and for complete fairness in case of the same values you can sort the list by experience (like at the auction). Such a system will stimulate the economy to a greater extent because players will have to turn in as many requests as possible.

Interesting point. So the prediction is that ppl will max out the restriction which still activates the trading.
I get what you are talking about now.

**Comment by alexandrelira99** [comment]
_"Change the chore structure to weekly and make them overloaded (No limit on chores for the week)"_

I particularly like this idea, but I would add a “window system” here.

Each day (24-hour cycle) the player could take part in one of the four competition times available in UTC:

12:00 AM - 06:00 AM
06:00 AM - 12:00 PM
12:00 PM - 6:00 PM
06:00 PM - 12:00 AM
...

Every day at 12:00 AM the list of chores would be updated and immediately the farmer could choose to take part in this “round” that will go on until 06:00 AM and during this time there will be a leaderboard for the completed chores, with of course a weighted score for the most complex chores.

**Reply by ShinKan42** [reply]
kindda nice. but kindda too intense.
I think the weekly is to solve the issue of afk irl.

**Reply by alexandrelira99** [reply]
> _"Change the chore structure to weekly and make them overloaded (No limit on chores for the week)"_
> 
> I particularly like this idea, but I would add a “window system” here.
> 
> Each day (24-hour cycle) the player could take part in one of the four competition times available in UTC:
> 
> 12:00 AM - 06:00 AM 06:00 AM - 12:00 PM 12:00 PM - 6:00 PM 06:00 PM - 12:00 AM ...
> 
> Every day at 12:00 AM the list of chores would be updated and immediately the farmer could choose to take part in this “round” that will go on until 06:00 AM and during this time there will be a leaderboard for the completed chores, with of course a weighted score for the most complex chores.

getting back to complement: Every "6 hours windows" for the race, change the chores so it won't be unfair with those who only can participate in the first "window" (from 12 AM to 6 AM).

I would like to also remember that it would bring more value to many SFTs, specially those related to time ("speed") and also temporary SFTs such as time warp totem and "hourglasses" from factions market.

**Comment by Thebyber** [comment]
I am a player who likes to be fighting for the best NFTs every season and I like to progress my bumpkin level and expand to always be on the latest expansion.

This season started badly when seeing the good NFTs and their low stock... since the drill or the shovel are very broken (I understand their low stock).. the gnome (if you are an advanced player and have plenty of oil) and dev wrench ( This one if you have plenty of iron if not it's meh) they are good nfts but with little stock... they should have 50 stock each for tickets. Then the oil overall is fine but I would have put more stock too. The rest of the nfts are acceptable that I prefer to buy them through opensea since their buffs are disappointing compared to the previous nfts.

Speaking of Stella's nfts, they are normal and I see it well... the NFT of 5k tickets should have a greater impact since it is not attractive to obtain it... the players who obtain it are because they could not get any of the nfts of auctions not because a good nft sees it.

Speaking of the beach NFTs, they are decent and I see them well.

After analyzing the NFTs... let's analyze the factions. Without a doubt, this ticket mechanic is being very good for burning resources and money for the players who want to compete for the auctions but it is too frustrating and not healthy... I am also from Spain and I have to be awake every Sunday at 2 to continue donating resources to be able to remain in the top 25... I have fought 3 times to try to be above the top 10 and I have only achieved it once with a top 5...

Furthermore, this mechanic seems abusive to me and does not allow me to advance in my bumpkin progress since I have to be doing all the missions and some do not plant the seeds that I prefer to plant or cook the most optimal dishes to level up in the bumpkin. Currently I can only focus on competing to stay above the top 25 and I can't afford to continue progressing my bumpkin level.

The best thing about the season is the return to the pirate island, I missed being able to dig. Also, the faction's mission system is entertaining and I quite like it... perhaps it is short on missions since they can all be completed in 4-5 days. I think Hank's missions should be like those of the factions... weekly and not daily.

Another thing I hate about this season is having to be frustrated cooking in order to compete by feeding the pet, since it is the most economical way to get marks. Why don't kitchens have a system like the crop machine integrated automatically? Oil could be used to have automatic food production. Why doesn't it integrate being able to cook two dishes at the same time? It is as easy as hiring a second chef in each kitchen (which could be the bumpkins that we have added to the house, I currently have 5 and they are for clothing... these bumpkins could be placed in the kitchens to be able to cook more dishes or even the same 2 times).

Speaking of the mini-games to get marks, I see it as good but very sacrificed... since it becomes very tiresome to play the 3 games every day... don't you think it's better to put a game every day and rotate it?  
Monday, Thursday and Saturday: Chicken Rescue
Tuesday and Thursday: Crops & Chickens
Wednesday and Sunday: Fruit Dash

In conclusion, I am a little burned out and frustrated this season since competing is too hard and does not allow me to advance my progress in my bumpkin since I have to have the kitchens active and then donate everything I cook...

The best: 
-I think the faction mission system is very healthy and bearable.
-The return of the pirate island
-I think the rework of the mascot is more logical than before.

The worst: 
-The season tickets in the ranking of the factions
-Hank's daily missions are boring and monotonous
-The low stock of good nfts in the season.
-Daily slavery in the minigames if you want to compete

**Comment by alexandrelira99** [comment]
![image](https://github.com/user-attachments/assets/1822c748-d92e-442e-8eec-52ffeb625c44)
I would add all basic, medium and advanced crops constantly all weeks.. in the current model it's causing constant pump & dump on prices. If we keep the resources constantly "donatable" we would keep their demand high all the time, with short periods of opportunity between them.

**Comment by fra-vh** [comment]
Make chores weekly, in a similar format to the faction ones (3 active, 1 skip per day). No ticket auctions on the weekly chores reset day to allow everyone to do some chores independently of their timezone.

the rest of the options don't seem good to me, the format is quite good now in my opinion

**Comment by Haesoo2299** [comment]
I am a small/mid-level player so I wanted to put down my perspective. I did enjoy this season and managed to get most of the items I had planned for. However, I am a tad let down by the use-case of the items. Say I get the tomato / lemon boosts, I want to see them in at least a few recipes in this season itself so that I can put them to use and not have to wait for a future season to see its potential. It would not only help me trade and get better value for the produce but will also help level my bumpkin faster. Apart from that I do understand that to get the best NFT, you gotta plan and grind accordingly so no complains there.

**Comment by Theadra** [comment]
We're late to the comments, but still sharing, I highly enjoyed this last season, my partner hated it. It was so exhausting and expensive, that we're sitting out being highly competitive in this next season to relax.  I personally liked the grind and very tight scores, keeping the season interesting every week, my partner felt overwhelmed every week.
